.gitignore

```diff
@@ -1,4 +1,6 @@
+coverage/
 dist/
 src/**/*.js
 src/**/*.d.ts
-*~
\ No newline at end of file
+*~
+node_modules/
```

Comment:
looks like this was added later. node modules are still in commit

Reply:
I think it would be good if we add `.npmignore` to ignore tests going into npm package

Reply:
Added .npmignore with **/__tests__/**/*